sceptre/stack.py

```diff
@@ -196,7 +196,8 @@ def create(self):
             "Tags": [
                 {"Key": str(k), "Value": str(v)}
                 for k, v in self.config.get("stack_tags", {}).items()
-            ]
+            ],
+            "OnFailure": self.config.get("on_failure", 'ROLLBACK')
```

Comment:
I feel `OnFailure` should only be added to the kwargs when its present in the stack config. To ensure AWS CloudFormation's default behaviour occurs (which could change from `ROLLBACK`) - the `OnFailure` should be absent from the API call.